format.py

```diff
@@ -0,0 +1,83 @@
+import re
+import sys
+from collections import defaultdict
+
+### BEFORE YOU RUN THIS SCRIPT, READ THE BELOW LINES
+### This script will look at a file that has the output of dt_binding_check (preferably all of the output. meaning stdout AND stderr) 
+### To have both stdout and stderr be printed into one file, you must run the following command: dt_binding_check &> dtvout
+### On line 27, the file that will be read is selected, please be sure that the name of the file that you intend to analyze is the first argument of the "open" command
+
+"""
+My Notes
+Things to make this script print: 
+	Status of a file (pass/fail/warning)
+		Right now, Trying to test the status of every line begining with "CHKDT" 
+			Done. Seems to be no problems
+		Warnings (line 1904 of dtvout) 
+			separated
+
+Things To Do:
+	General printing of the files that passed and failed
+
+Things Done: 
+	Separated Files with Errors or warnings from the rest
+
+"""
+
+# if len(sys.argv) == 1: #we need an argument to identify the name of the file
+# 	print("no arguments")
+# 	exit()
+
+dtoutput = open("dtvout", "r")# the second argument (aka argv[1]) will be the file that we look at ## sys.argv[1]
```

Comment:
Erm, yes as your comment says it would be better to parse command line arguments.  I think it's fine for this first version to have hard-coded `dtvout` as the file name, but another follow-up would be to clean up the script using a `main()` function and `argparse` to parse the command line.